qt/scientific_interfaces/test/ISISReflectometry/MainWindow/MainWindowPresenterTest.h

```diff
@@ -68,6 +69,15 @@ class MainWindowPresenterTest : public CxxTest::TestSuite {
     verifyAndClear();
   }
 
+  void testOptionsDialogPresenterSubscribesToMainWindow() {
```

Comment:
I think the name is misleading here?
```suggestion
  void testMainWindowPresenterSubscribesToOptionsDialogPresenter() {
```

Reply:
Yes, got myself confused!